docs/bridge/architecture.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Bridge Architecture
+sidebar_position: 3
+sidebar_label: Bridge Architecture
+---
+
+# Bridge Architecture
+
+## Overview
+This page provides a technical overview of the Vara Bridge architecture, without covering every component in exhaustive detail. The goal is to give readers a high-level understanding of how the Bridge operates. For complete technical information, please refer to the [Gear Bridges repository](https://github.com/gear-tech/gear-bridges).
+
+Vara Bridge uses Plonky2 circuits to implement cryptographic primitives, forming a secure foundation for cross-chain operations. Plonky2 is a zero-knowledge (ZK) proof system that provides efficient and scalable proofs. By using these circuits, Vara Bridge preserves transaction integrity and greatly reduces the possibility of manipulation. The proof mechanism also guarantees that specific pallet states are finalized and included in the Vara Network before those states are relayed to Ethereum. Verified within a Gnark circuit, these proofs help maintain trust and accuracy as data moves across chains.
+
+## Bridge Components
+The Vara Bridge consists of several distinct parts, including on-chain components on both Vara and Ethereum, as well as off-chain components for securely relaying message proofs.
+
+### Components on Vara Network
+
+- **VFT programs**: Deployments of the VFT token standard, capable of transferring, burning, and minting VFT tokens. Each unique token has its own VFT standard deployment on Vara.
```

Comment:
its own VFT standard? sounds like they are based on different standards. They all are implemented via the same standard. And the link would be useful: /docs/build/standards/vft